Week1/Section2/Exercise4/Source4.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+Exercise 4 - True or False
+Write a quiz program for the user.
+The program should ask a series of true or false questions to the user.
+For each answer, tell the user if they were correct or incorrect.
+At the end of the quiz, tell them their final grade (grade should be 0-5)
+
+Example output
+Welcome to your quiz!
+0 - False
+1 - True
+
+Question 1: Are avocados fruits? 0
+WRONG! How could you miss that??
+
+Question 2: Is the sky blue? 1
+CORRECT! Looks like you have common sense.
+
+Question 3: Is Teravision the Best.Team.Ever? 1
+CORRECT! Looks like you have common sense.
+
+You got 2/3 correct answers! Your final grade is: 3.33
+*/
+#include <iostream>
+
+int main() {
+	bool answers[3];
+	float grade = 0;
```

Comment:
Remember that floats should end with an f. `0.f` or `0.0f` is more appropriate here. Otherwise, you're assigning an int to a float.